src/reducers/application.js

```diff
@@ -72,6 +97,21 @@ export default (state = initialState, action) => {
         menuLinks,
       };
     }
+    case MENU_FILTERED:
```

Comment:
Is there a particular reason why we have to store the computed data in the store? If not, we could probably do this on run time while getting the data. Here's link for the redux docs which should provide some idea what could be done https://redux.js.org/recipes/computing-derived-data

Reply:
@lauriii I followed the approach @mattgrill was doing on the permissions page. I think having less state aka. computing the values as you go, is the better approach. If there are performance issues with it, we could always come back to it. 👍

Reply:
Yeah. The permissions page needs some work right now since it's still not using the saga/redux pattern. At that migration point it would be nice to reduce the JSON API payload to just the items we need.

Reply:
Did we agreed on using redux for storing the filter string for now?

Reply:
If it's not too much work, I think it would be better to move the string to the local state :)